app.js

```diff
@@ -0,0 +1,209 @@
+console.log('hello from app.js')
+
+// select DOM elements
+const squares = document.querySelectorAll('.square')
+const startBtn = document.querySelector('.startBtn')
+const resetBtn = document.querySelector('.resetBtn')
+const p12 = document.querySelectorAll('.player')
+const p1 = document.querySelector('#p1')
+const p2 = document.querySelector('#p2')
+const pScores = document.querySelectorAll('.pScore')
+const gamesPlayed = document.querySelector('#gamesPlayed')
+
+
+const players = [
+    {
+        player: 1,
+        myTurn: false,
+        char: 'X',
+        wins: 0
+    },
+    {
+        player: 2,
+        myTurn: false,
+        char: 'O',
+        wins: 0
+    }
+]
```

Comment:
Good use of an object but this could also be simplified by setting a boolean to check if it's player 1's turn and if not then it can be assumed it is player 2's turn. This would clean up a few of your other lines of code where you're explicitly defining X and O